runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -1045,6 +1045,9 @@ class CompilationInfo
    bool              isInShutdownMode() {return _isInShutdownMode;}
    void              setIsInShutdownMode() {_isInShutdownMode = true;}
 
+   bool              isSwapMemoryDisabled() {return _isSwapMemoryDisabled;}
+   void              setIsSwapMemoryDisabled() {_isSwapMemoryDisabled = true;}
```

Comment:
This setter should allow to set `_isSwapMemoryDisabled` to both `true` or `false`. See below why that is needed.